docs/en/query_language/functions/hash_functions.md

```diff
@@ -64,5 +64,52 @@ A fast, decent-quality non-cryptographic hash function for a string obtained fro
 `URLHash(s, N)` – Calculates a hash from a string up to the N level in the URL hierarchy, without one of the trailing symbols `/`,`?` or `#` at the end, if present.
 Levels are the same as in URLHierarchy. This function is specific to Yandex.Metrica.
 
+## farmHash64
+
+Calculates SipHash from a string.
+Accepts a String-type argument. Returns UInt64.
+For more information, see the link: <https://github.com/google/farmhash>
```

Comment:
you could make name of hash a link to description instead of providing it separately